CLASSES/class_person.py

```diff
@@ -0,0 +1,92 @@
+class Person:
+    max_health = 100
+    base_stats = ("attack", "defense", "intelligence", "health")
```

Comment:
To jest do wywalenia. Nie ma sensu żeby każda instancja tej klasy trzymała tą tuple u siebie